LICENSE

```diff
@@ -1,21 +0,0 @@
-The MIT License (MIT)
```

Comment:
If their code isn't in our repo anymore, we don't have to keep their
license there any more either, though if we want to be good custodians of
our code/project we should document that "this project also uses the
following things under the following licenses" underneath our license for
all our top-level dependencies.   The license terms of the dependencies
still apply though, and whatever license we choose needs to be compatible
with them (i.e. no GPL licensing our stuff if it's using MIT libraries,
etc).  There's a little more wiggle room  in our case because we don't
distribute all the dependencies and it's up to the user to download them
(via pip etc), but regardless I would rather stay with a license that meets
both the spirit and the letter of the licensing terms, to be good
open-source citizens.



On Wed, Aug 19, 2015 at 9:43 PM, Tom Dooner <notifications@github.com>
wrote:

> In LICENSE
> <https://github.com/opencalifornia/disclosure-backend/pull/16#discussion_r37494669>
> :
>
> > @@ -1,21 +0,0 @@
> > -The MIT License (MIT)
>
> Not sure if we're legally required to keep it since we originally had the
> code here... We shouldn't need the license if we're just pulling
> calaccess_raw in as a dependency (though the code in example/ is still
> from upstream). Maybe it does make sense to start the git repo anew.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/opencalifornia/disclosure-backend/pull/16/files#r37494669>
> .
>